pages/_app.js

```diff
@@ -0,0 +1,12 @@
+import Layout from "../Components/Layout";
+import "../styles/globals.css";
+
+export const MyApp = ({ Component, pageProps }) => {
```

Comment:
You're exporting it as a const and a default export.

I think you might only need export default MyApp and not export const MyApp

Try changing line 4 to const MyApp

Reply:
Difference:

When you export it as
`export const MyApp = () => {}`
You import it as
`import { MyApp } from './MyApp'`

When you export it as default
`export default MyApp`

You import it as:
`import MyApp from './MyApp'`